File: SiteSpeedManager.Master/Resources/V1/Country.cs
﻿using Glyde.Web.Api.Resources;

namespace SiteSpeedManager.Master.Resources.V1
{
    [Resource("countries")]
    public class Country : Resource<string>
    {
        public string DisplayName { get; set; }

        public bool IsEnabled { get; set; }
    }
}

File: UI/src/app/models/country.ts
export class Country {
    public id: string;
    public displayname: string;
    public isEnabled: boolean;

    constructor (id: string, displayname: string, isEnabled: boolean)
    {
        this.id = id;
        this.displayname = displayname;
        this.isEnabled = isEnabled;
    }
}
